config/settings.js

```diff
@@ -31,6 +31,7 @@ const SCHEMA_PIPELINE_SETTINGS = Joi.object()
         groupedEvents: Joi.boolean().optional(),
         showEventTriggers: Joi.boolean().optional(),
         filterEventsForNoBuilds: Joi.boolean().optional(),
+        filterSchedulerEvents: Joi.boolean().optional(),
```

Comment:
what's default here ? it should how scheduler events by default.

Reply:
yes,by default it should be false